conman/routes/managers.py

```diff
@@ -30,3 +30,17 @@ def best_match_for_path(self, path):
         except IndexError:
             msg = 'No matching Route for URL. (Have you made a root Route?)'
             raise self.model.DoesNotExist(msg)
+
+    def move_branch(self, old_url, new_url):
+        """
+        Move a Route and all descendants to a new URL.
```

Comment:
Come to think of it... it might be worth considering some kind of validation on these, otherwise I might be making a bit of a foot-gun here.